2-mandatory/1-create-functions.js

```diff
@@ -44,7 +52,11 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(arr) {
+    return arr
+      .map((x) => (x > 100 ? "100" : x.toFixed(2)))
+      .map((y) => `${+y}%`);
```

Comment:
nice use of a ternary here, although you can do both of these `.map` operations in a single `.map` to save going through the array twice; is more efficient and easier to read